js/sykmeldinger/sykmelding-ny/sykmelding-skjema/skjema/DinSykmeldingSkjema.js

```diff
@@ -162,6 +163,19 @@ export class DinSykmeldingSkjemaComponent extends Component {
                         </div>
                     )}
                 />
+                <Vis
+                    hvis={sykmelding.mulighetForArbeid.perioder[0].behandlingsdager !== null
+                        && (brukersSvarverdier.valgtArbeidssituasjon === ARBEIDSTAKER
```

Comment:
kommer det kun til å være én periode i en slik sykmelding?

Reply:
Den kan fort være flere perioder i 1 sykemelding med flere behandlingsdsger, men dei kan ikkje være meir enn 1 pr uke. Det har vi regelen for FOR_MANGE_BEHANDLINGSDAGER_PER_UKE

Reply:
Sjekker nå alle perioder